lambdas/src/apis/collections/controllers/wearables.ts

```diff
@@ -120,7 +123,18 @@ export async function getWearables(
     }
   }
 
-  return paginatedOffChainWearables.concat(onChainWearables)
+  let moreData: boolean = false
+  if (offChainWearables.length > (onChainPagination.offset + 1) * onChainPagination.limit) {
+    moreData = true
+  }
+
+  const allPagination = { ...onChainPagination, moreData: moreData }
+
+  return {
```

Comment:
we can't break backwards compatibility with people using this endpoint. I understand it changed the schema of the response. 

We should have another new endpoint to handle this filtered case.

Reply:
This endpoint has not been released, nor has it been deployed to a dev catalyst. We can still make breaking changes

Reply:
Thanks Nico